services/QuillLMS/app/models/question.rb

```diff
@@ -149,4 +154,37 @@ def rematch_type
   private def stored_as_array(key)
     data[key].class == Array
   end
+
+  private def validate_sequences
+    return if !data.present? || !data.is_a?(Hash)
+    parse_and_validate(data['incorrectSequences'])
+    parse_and_validate(data['focusPoints'])
+  end
+
+  private def parse_and_validate(sequences)
+    return if !sequences.present?
+    if sequences.is_a?(Hash)
+      sequences.each do |key, value|
```

Comment:
A one-liner block is fine when you only have a short statement:  
`sequences.each { |key, value| validate_text_and_feedback(value) }`